src/App.js

```diff
@@ -2,15 +2,20 @@ import { Routes, Route } from "react-router-dom";
 import "./App.css";
 import Header from "./Components/Header";
 import Nav from "./Components/Nav";
-
+import ArticleByTopic from "./Components/ArticleByTopic";
 import AllArticles from "./Components/AllArticles";
 function App() {
   return (
     <div className="App">
       <Header />
       <Nav />
       <Routes>
-        <Route path="/articles" element={<AllArticles />} />
+        <Route
+          path="/articles"
+          element={<AllArticles />}
+          AllArticles={AllArticles}
```

Comment:
Line 16 wouldn't be needed, all we require in \<Route>\</Route> is the path and element :)